Sources/Contour/Helpers/Hasher.cs

```diff
@@ -1,39 +0,0 @@
-﻿using System;
-
-using Newtonsoft.Json;
-
-namespace Contour.Helpers
-{
-    /// <summary>
-    /// Рассчитывает значение хеш-функции для содержимого.
-    /// </summary>
-    public class Hasher
```

Comment:
Breaking backward compatibility. You should avoid it.

Reply:
Reverted deletion and marked as obsolete.